### ISSUE TITLE
Application crashed when solving thousands of subjects

### ISSUE THREAD

**Issue by POEG1726** [Lymphatus/caesium-image-compressor#154]
**Describe the bug**
Cannot solve roo many subjects
Most of them are about 20MB
File format:   .jpg
**Software version**
Version2.2.1

**Operating System information**
 - OS: Windows10
 - Version 19044.1889

**To Reproduce**
1. Go to 'Add the whole dir(path could exist Chinese&Korean,but i think it is okay)'
2. Click on 'quality80&keep dir structure&start'
4. See error in about 4000th subject

**Expected behavior**
Application crashed&window disappeared

**Screenshots**
sorry,i use that in virtual machine and run that take me long time,i do not want to do it again,the behavior happened so quickly that i cannot snap.

**Application Log**
!!I have omitted many seemingly similar things!!
```
[2022-09-18 05:10:30.245][I] ---- Closing application ----
[2022-09-18 09:30:53.426][I] ---- Starting application ----
[2022-09-18 09:30:53.488][I] Starting UI
[2022-09-18 09:32:18.082][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/GtBk5oHsTH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:32:18.129][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/IPJBmfYrrU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:32:18.176][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/of8PYgSnip.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:39:14.271][W] "Z:/download/bp_photos/����[Espacia Korea] EXC#056 KANA/iMencCP7Zm.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:06.865][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/6PvHs3FTrp.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:06.896][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/6s5YJCwysK.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:07.068][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/b1KKAVkINU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:07.256][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/DwlU59hwZC.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:07.818][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/tKVuB33hwQ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:40:07.959][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/ZOUaQ3VBqR.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.678][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/5ahKGNXGcF.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.710][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/5rFK9bOifi.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.804][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/74f4vbyaMh.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.850][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/79MMq0nJm5.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.866][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/9CiS7FxDca.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.913][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/9Fll2E6Tew.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:24.960][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/a2ydGIIwJx.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.007][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/fo5oiCmbQN.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.038][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/gMWb2pWVD2.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.085][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/h9mQVCOzrr.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.116][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/hLjZHKHjsf.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.147][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/jIRLBhMoP5.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:43:25.272][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Jvd3bQ9uxe.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.163][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Ov3HMrcwyw.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.194][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/qQdlk7Vu4j.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.225][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Uw3jihxswU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.272][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/UX1Ch0cVbN.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.319][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/ve53RNOjKR.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.335][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/vWD0x0pX20.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.413][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/WqBghZcujj.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.444][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/XGpYld71sA.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.491][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/y0cnwZDp6n.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.569][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/YgZ2zKO4AJ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.585][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/zocCw7KTSO.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:44:09.600][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/ZtaaoA9wUz.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:34.873][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/0lOq6UlDsE.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:35.060][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/6TNfyBMM89.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:35.217][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/9vzd7NreUh.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:35.326][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/Dn5thnNsqu.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:35.795][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/pP4VwqTjvL.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:35.936][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/VRJsiw49ul.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:56.473][W] "Z:/download/bp_photos/���S���涬 - NO.106 ��������Ů���ϩ`�ե����ࡷ[150P2V-565MB]/eubZtKPCaU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:47:56.583][W] "Z:/download/bp_photos/���S���涬 - NO.106 ��������Ů���ϩ`�ե����ࡷ[150P2V-565MB]/FFVopH4HM0.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:48:11.145][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/0dvJA4J1en.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:48:13.395][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/KWwDvNz7re.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:48:13.474][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/LLgjHvdxOH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:48:13.754][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/MSGwvb328D.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:49:14.381][W] "Z:/download/bp_photos/Bambi?? - NO.18 Lacerta Photography [101P-473MB]/aNEWeNcXA1.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:49:35.850][W] "Z:/download/bp_photos/�O���� - [Loozy] Librarian Girl [84P2V-1.75GB]/cMthasTtNt.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:49:37.881][W] "Z:/download/bp_photos/�O���� - [Loozy] Librarian Girl [84P2V-1.75GB]/naa9dMJaIZ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:54:05.300][W] "Z:/download/bp_photos/����Jean - NO.73 �����ķ�ɫ��Ӿ [60P2V-365MB]/osoum5MGJv.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:54:05.831][W] "Z:/download/bp_photos/����Jean - NO.73 �����ķ�ɫ��Ӿ [60P2V-365MB]/zkGgP486el.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:54:58.924][W] "Z:/download/bp_photos/��֭è�� - NO.80 ������Ů [60P1V-1.73GB]/bnau5BHX61.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 09:56:09.626][W] "Z:/download/bp_photos/˫��β��Ů���߽� - NO.65 ����ë�� [63P-50MB]/bJh1tbUwE6.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 10:00:57.548][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/339f1zEBRB.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 10:00:57.876][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/Bh8qgB3ReQ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 10:00:57.938][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/G2iJXkvkBj.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 10:00:58.500][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/mzEGN5UoNH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 11:15:50.145][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\746ebc29ace1fd9cb11197571576642912cc566999dff9ba3a4166fe182e68f0"
[2022-09-18 11:15:50.145][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\43d651e450b55b84f4249e333e037cc2d88008c1444b6b545771cd7f7ff8202a"
[2022-09-18 11:15:50.145][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\a982f5e6503d0c25437949702c6c2cb41088dd9a237f5d6629a30bf557d05d20"
[2022-09-18 11:15:50.145][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\914bf2c771369c5aa592b2d7285bea581ce7f5364411cf260f5c61f1414825a2"
[2022-09-18 11:15:50.161][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\552a04997b32c5b45dae2aa76e907149946fb3574051bf85d141667e56d5df51"
[2022-09-18 11:15:51.301][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\73e4ba099311f09f74f04ebdfc4a48d1191a4d549649c0b30d4011a6dadfec42"
[2022-09-18 11:15:51.457][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\2fb1f4257aedcb07b9bb50078a865db8922304554982a1733d23c61511c35a1d"
[2022-09-18 11:15:51.504][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\89bacf06873b9711f040ec358e34e8ffed9f55141290e858e61c05dcaabd90a8"
[2022-09-18 11:47:23.487][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\1d522be1b21605cf00daa042d2fd4192e6c9aceeb1deb07358433666ae4bbe6d"
[2022-09-18 11:47:30.380][W] QObject::killTimer: Timers cannot be stopped from another thread
[2022-09-18 11:47:30.438][W] QObject::~QObject: Timers cannot be stopped from another thread
[2022-09-18 11:57:52.145][I] ---- Starting application ----
[2022-09-18 11:57:52.238][I] Starting UI
[2022-09-18 11:59:57.130][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/GtBk5oHsTH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 11:59:57.177][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/IPJBmfYrrU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 11:59:57.223][W] "Z:/download/bp_photos/����??? Addielyn �C (ad1yn2) [AfreecaTV] �C ����Ӿ�� (2021.06.30) [0P3V-11.1GB]/of8PYgSnip.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:11:27.967][W] "Z:/download/bp_photos/����[Espacia Korea] EXC#056 KANA/iMencCP7Zm.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:32.928][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/6PvHs3FTrp.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:33.022][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/6s5YJCwysK.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:33.272][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/b1KKAVkINU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:33.459][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/DwlU59hwZC.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:33.959][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/tKVuB33hwQ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:12:34.194][W] "Z:/download/bp_photos/��֭è��  ��ѧ������ [38P6V-401MB]/ZOUaQ3VBqR.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:36.873][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/5ahKGNXGcF.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:36.966][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/5rFK9bOifi.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:36.998][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/74f4vbyaMh.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.029][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/79MMq0nJm5.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.060][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/9CiS7FxDca.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.122][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/9Fll2E6Tew.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.154][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/a2ydGIIwJx.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.216][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/fo5oiCmbQN.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.247][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/gMWb2pWVD2.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.279][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/h9mQVCOzrr.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.310][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/hLjZHKHjsf.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.388][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/jIRLBhMoP5.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:16:37.466][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Jvd3bQ9uxe.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.253][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Ov3HMrcwyw.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.269][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/qQdlk7Vu4j.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.300][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/Uw3jihxswU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.347][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/UX1Ch0cVbN.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.410][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/ve53RNOjKR.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.441][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/vWD0x0pX20.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.503][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/WqBghZcujj.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.519][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/XGpYld71sA.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.581][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/y0cnwZDp6n.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.644][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/YgZ2zKO4AJ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.675][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/zocCw7KTSO.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:17:33.722][W] "Z:/download/bp_photos/����leehee������ ģ�� Jeon Ji Su(G-su)/ZtaaoA9wUz.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.055][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/0lOq6UlDsE.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.227][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/6TNfyBMM89.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.336][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/9vzd7NreUh.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.477][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/Dn5thnNsqu.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.758][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/pP4VwqTjvL.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:17.899][W] "Z:/download/bp_photos/��֭è�� - ��Ů���� [35P6V-829MB]/VRJsiw49ul.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:38.118][W] "Z:/download/bp_photos/���S���涬 - NO.106 ��������Ů���ϩ`�ե����ࡷ[150P2V-565MB]/eubZtKPCaU.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:38.228][W] "Z:/download/bp_photos/���S���涬 - NO.106 ��������Ů���ϩ`�ե����ࡷ[150P2V-565MB]/FFVopH4HM0.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:53.369][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/0dvJA4J1en.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:55.181][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/KWwDvNz7re.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:55.212][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/LLgjHvdxOH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:21:55.415][W] "Z:/download/bp_photos/��٤�� - �ȶ�����2/MSGwvb328D.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:22:52.744][W] "Z:/download/bp_photos/Bambi?? - NO.18 Lacerta Photography [101P-473MB]/aNEWeNcXA1.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:23:13.525][W] "Z:/download/bp_photos/�O���� - [Loozy] Librarian Girl [84P2V-1.75GB]/cMthasTtNt.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:23:15.540][W] "Z:/download/bp_photos/�O���� - [Loozy] Librarian Girl [84P2V-1.75GB]/naa9dMJaIZ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:27:47.352][W] "Z:/download/bp_photos/����Jean - NO.73 �����ķ�ɫ��Ӿ [60P2V-365MB]/osoum5MGJv.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:27:48.164][W] "Z:/download/bp_photos/����Jean - NO.73 �����ķ�ɫ��Ӿ [60P2V-365MB]/zkGgP486el.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:28:45.321][W] "Z:/download/bp_photos/��֭è�� - NO.80 ������Ů [60P1V-1.73GB]/bnau5BHX61.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:29:51.801][W] "Z:/download/bp_photos/˫��β��Ů���߽� - NO.65 ����ë�� [63P-50MB]/bJh1tbUwE6.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:34:39.129][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/339f1zEBRB.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:34:39.661][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/Bh8qgB3ReQ.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:34:39.770][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/G2iJXkvkBj.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:34:40.270][W] "Z:/download/bp_photos/��֭è�� - ��ݮʥ��[45P4V-0.99GB]/mzEGN5UoNH.jpg" is not supported. Error: Image format is not supported.
[2022-09-18 12:38:27.098][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\8200d4546ee2168dba2dd6bfd103e484dce90488a325938cc818253aa9770e2b"
[2022-09-18 12:38:27.098][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\96cfc3becedbea8d6c090ef979af28bf19abe18d0efdee6c6d1adb1f43847b0e"
[2022-09-18 12:38:27.098][I] Preview in "C:/Users/admin/AppData/Local/SaeraSoft/Caesium Image Compressor/cache\\7e8c2a7ad3c493d3132dea1e7890dc76a6c8d33b3e8c6e8d8251ded1906375ba"
[2022-09-18 13:09:03.520][W] QObject::killTimer: Timers cannot be stopped from another thread
[2022-09-18 13:09:03.598][W] QObject::~QObject: Timers cannot be stopped from another thread
[2022-09-18 13:14:14.113][I] ---- Starting application ----
[2022-09-18 13:14:14.192][I] Starting UI
[2022-09-18 13:24:11.958][I] ---- Closing application ----
```

**Comment by POEG1726** [comment]
i will upload logs later

**Comment by Lymphatus** [comment]
Did you try without the Virtual Machine?
Caesium is pretty heavy when compressing a log of pictures this may consume all the Virtual Machine resources.
You can also go to Preferences -> General -> Deselect the "Multithreading" option. The process will be longer, but can solve a lot of issues on those enviroments.

**Comment by POEG1726** [comment]
> 

i have deselect the "Multithreading" option,but it crashed again in about the 13kth subject,and the last line in log is
```
[2022-09-24 09:26:24.560][W] QObject::killTimer: Timers cannot be stopped from another thread
[2022-09-24 09:26:24.622][W] QObject::~QObject: Timers cannot be stopped from another thread
```
It seems to be the same every time,whether the "Multithreading" option is selected or not
by the way,the Subjects are on SMB server
but i think the SMB server is extremly stable,i have run many automatic tasks on it just like CAESIUM

**Comment by POEG1726** [comment]
> Did you try without the Virtual Machine? Caesium is pretty heavy when compressing a log of pictures this may consume all the Virtual Machine resources. You can also go to Preferences -> General -> Deselect the "Multithreading" option. The process will be longer, but can solve a lot of issues on those enviroments.

> 

i have deselect the "Multithreading" option,but it crashed again in about the 13kth subject,and the last line in log is
```
[2022-09-24 09:26:24.560][W] QObject::killTimer: Timers cannot be stopped from another thread
[2022-09-24 09:26:24.622][W] QObject::~QObject: Timers cannot be stopped from another thread
```
It seems to be the same every time,whether the "Multithreading" option is selected or not
by the way,the Subjects are on SMB server
but i think the SMB server is extremly stable,i have run many automatic tasks on it just like CAESIUM

**Comment by Lymphatus** [comment]
I think the SMB server might be the problem. I've not run extended tests on this kind of drive, but I had other users reporting crashes or instabilities. I need to investigate further and discover the cause of this problems, but it's not an easy environment to reproduce and might take some time.

**Comment by POEG1726** [comment]
> I think Caesium running on a SMB server might be the problem. I've not run extended tests on this kind of drive, but I had other users reporting crashes or instabilities. I need to investigate further and discover the cause of this problems, but it's not an easy environment to reproduce and might take some time.

got it.
I run CAESIUM for 24000 subjects without problem,on my PC with multithreading, instead of Virtual Machine,everything is normal now.
So i think Virtual Machine is the main reason!

**Comment by rudolphos** [comment]
I tried opening 256 MB image file and the app didn't open it at all. No crash though. Version v2.2.1
![image](https://user-images.githubusercontent.com/4366810/210894431-8e3fe0ef-cbb9-4def-b15a-ed4f4dd23a91.png)

**Comment by Lymphatus** [comment]
@rudolphos maximum file size allowed is 200Mb.

**Comment by jl0476** [comment]
Same issue when I compress 10000 pics
I tried to compress files in SMB

**Comment by Lymphatus** [comment]
Fixed in 2.4.0
